openstates/ky/committees.py

```diff
@@ -41,14 +41,43 @@ def scrape(self, chamber, term):
                 links = links + linkz
 
             for link in links:
-                name = re.sub(r'\s+\((H|S)\)$', '', link.text).strip().title()
-                name = name.replace(".", "")
-                comm = Committee(chamber, name)
-                comm_url = link.attrib['href'].replace(
-                    'home.htm', 'members.htm')
-                self.scrape_members(comm, comm_url)
-                if comm['members']:
-                    self.save_committee(comm)
+                self.scrape_committee(chamber, link)
+                
+
+    def scrape_committee(self, chamber, link, parent_comm=None):
+        home_link = link.attrib["href"]
+        name = re.sub(r'\s+\((H|S)\)$', '', link.text).strip().title()
+        name = name.replace(".", "")
+        if "Subcommittee " in name:
+            subcomm = name.split("Subcommittee")[1]
```

Comment:
I'm removing the subcommittee part, but it looks like those aren't actual real subcommittees - they don't seem to report to anyone. So Sub. or Subcommittee or Comm. or whatever are gone, but they aren't listed as anyone's subcommittee. I think at least one of them reports to an executive department instead. Seem reasonable?